core/client/src/backend.rs

```diff
@@ -26,6 +26,11 @@ use state_machine::ChangesTrieStorage as StateChangesTrieStorage;
 use consensus::well_known_cache_keys;
 use hash_db::Hasher;
 use trie::MemoryDB;
+use parking_lot::Mutex;
+
+lazy_static! {
+    static ref IMPORT_LOCK: Mutex<()> = Mutex::new(());
```

Comment:
I'm not pushing, but let's probably remove default impl? There aren't many backend implementations (3 afaik && I'm almose sure we could use `unimplemented()` for in-mem backend). Otherwise all `Client` instances will be using the same lock. And we create several client instances in many tests (network, integration, ...). What do you think?